test/http-server/bestpractice/doctype4.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Check the doctype
+  </title>
+</head>
+<body>
+  <body>
```

Comment:
Please remove the duplicated body tag here. The doctype.html also has this issue, it'd be great to fix that as well.